### PR TITLE
Update trunc_normal_

### PR DESCRIPTION
**Pull request by liuruiyang98**
Remove jt.no_grad() to make var.require_grad = True
And modify jt.init.uniform_(var, XXX,XXX) to var.uniform_(XXX,XXX)

### CODE REVIEW

File: python/jittor/init.py [hunk 1]
```diff
@@ -706,26 +706,26 @@ def norm_cdf(x):
...
+    u = norm_cdf((b - mean) / std)
+
+    # Uniformly fill tensor with values from [l, u], then translate to
+    # [2l-1, 2u-1].
+    # var.uniform(2 * l - 1, 2 * u - 1)
+    var.uniform_(low=2 * l - 1, high=2 * u - 1)
+
+    # Use inverse cdf transform for normal distribution to get truncated
+    # standard normal
+    var.erfinv()
```
Comment (Jittor) [inline python/jittor/init.py:723]: erfinv is not a inplace function, change to var = var.erfinv()

File: python/jittor/init.py [hunk 2]
```diff
@@ -706,26 +706,26 @@ def norm_cdf(x):
...
+
+    # Transform to proper mean, std
+    var.multiply(std * math.sqrt(2.))
```
Comment (Jittor) [inline python/jittor/init.py:726]: same as above

File: python/jittor/init.py [hunk 3]
```diff
@@ -706,26 +706,26 @@ def norm_cdf(x):
...
+    # var.uniform(2 * l - 1, 2 * u - 1)
+    var.uniform_(low=2 * l - 1, high=2 * u - 1)
+
+    # Use inverse cdf transform for normal distribution to get truncated
+    # standard normal
+    var = var.erfinv()
+
+    # Transform to proper mean, std
+    var = var.multiply(std * math.sqrt(2.))
+    var.add(mean)
```
Comment (Jittor) [inline python/jittor/init.py:727]: add, clamp are not inplace function too, inplace function will end with underscore

**Review by liuruiyang98 — COMMENTED** [review]
modify to var = var.add(XX) and var = var.clamp(XX)

File: python/jittor/init.py (hunk 3 above)
Comment (liuruiyang98) [reply python/jittor/init.py]: Ok, it is done!

**Pull request merged**